exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -65,15 +68,13 @@ func newCwLogsPusher(expConfig *Config, params component.ExporterCreateSettings)
 		return nil, err
 	}
 
-	pusher := cwlogs.NewPusher(aws.String(expConfig.LogGroupName), aws.String(expConfig.LogStreamName), *awsConfig.MaxRetries, *svcStructuredLog, params.Logger)
-
 	logsExporter := &exporter{
-		svcStructuredLog: svcStructuredLog,
-		Config:           expConfig,
-		logger:           params.Logger,
-		retryCount:       *awsConfig.MaxRetries,
-		collectorID:      collectorIdentifier.String(),
-		pusher:           pusher,
```

Comment:
`pusher` is no longer being defined here, is there a configuration option that this would break?

Reply:
Pusher was always programmatically defined.  I found no cases where it was set custom.